packages/ui/src/memberships/modals/UpdateMembershipModal/UpdateMembershipModal.tsx

```diff
@@ -8,12 +8,15 @@ import { SignTransactionModal } from '@/common/modals/SignTransactionModal/SignT
 import { UpdateMembershipModalCall } from '@/memberships/modals/UpdateMembershipModal/index'
 import { createBatch } from '@/memberships/modals/UpdateMembershipModal/utils'
 
+import { useMyMemberships } from '../../hooks/useMyMemberships'
+
 import { updateMembershipMachine } from './machine'
 import { UpdateMembershipFormModal } from './UpdateMembershipFormModal'
 import { UpdateMembershipSuccessModal } from './UpdateMembershipSuccessModal'
 
 export const UpdateMembershipModal = () => {
   const { api } = useApi()
+  const { members, setActive, active } = useMyMemberships()
```

Comment:
Please remove unused variables:
```suggestion
  const { setActive } = useMyMemberships()
```